.github/ISSUE_TEMPLATE/meta_repository.yml

```
@@ -0,0 +1,37 @@
+name: "📇 <feature name>"
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["📇 Meta/Repository"]
+body:
+  - type: markdown
+    attributes:
+      value: '## Thanks for taking the time to contribute to the repository.'
```

I don't think it works with multiple value fields here. Try this instead
```suggestion
      value: '## Thanks for taking the time to contribute to the repository.'
  
  - type: markdown
    attributes:   
```

So the whole code snippet should look like this:
```yml
  - type: markdown
    attributes:
      value: '## Thanks for taking the time to contribute to the repository.'
  
  - type: markdown
    attributes:
      value: '## Before you continue, please search our open/closed issues to see if a similar issue has been addressed.'
```

---

ohhh that makes a lot of sense thank for the tip